go.mod

```diff
@@ -0,0 +1,66 @@
+module github.com/projectcalico/libcalico-go
+
+go 1.12
+
+require (
+	github.com/Azure/go-autorest v10.6.2+incompatible // indirect
+	github.com/beorn7/perks v0.0.0-20180321164747-3a771d992973 // indirect
+	github.com/coreos/bbolt v1.3.3 // indirect
+	github.com/coreos/etcd v3.3.8+incompatible
+	github.com/coreos/go-semver v0.3.0
+	github.com/coreos/go-systemd v0.0.0-20190620071333-e64a0ec8b42a // indirect
+	github.com/coreos/pkg v0.0.0-20180928190104-399ea9e2e55f // indirect
+	github.com/dgrijalva/jwt-go v3.0.0+incompatible // indirect
+	github.com/go-playground/locales v0.12.1 // indirect
+	github.com/go-playground/universal-translator v0.0.0-20170327191703-71201497bace // indirect
+	github.com/golang/groupcache v0.0.0-20190702054246-869f871628b6 // indirect
+	github.com/google/btree v0.0.0-20161005200959-925471ac9e21 // indirect
+	github.com/googleapis/gnostic v0.0.0-20170729233727-0c5108395e2d // indirect
+	github.com/gophercloud/gophercloud v0.0.0-20180330165814-781450b3c4fc // indirect
+	github.com/gorilla/websocket v1.4.0 // indirect
+	github.com/gregjones/httpcache v0.0.0-20170728041850-787624de3eb7 // indirect
+	github.com/grpc-ecosystem/go-grpc-prometheus v1.2.0 // indirect
+	github.com/grpc-ecosystem/grpc-gateway v1.9.4 // indirect
+	github.com/hashicorp/golang-lru v0.0.0-20160207214719-a0d98a5f2880 // indirect
+	github.com/imdario/mergo v0.0.0-20141206190957-6633656539c1 // indirect
+	github.com/jonboulle/clockwork v0.1.0 // indirect
+	github.com/kelseyhightower/envconfig v0.0.0-20180517194557-dd1402a4d99d
+	github.com/leodido/go-urn v0.0.0-20181204092800-a67a23e1c1af // indirect
+	github.com/matttproud/golang_protobuf_extensions v1.0.1 // indirect
+	github.com/onsi/ginkgo v1.6.0
+	github.com/onsi/gomega v1.4.2
+	github.com/pborman/uuid v0.0.0-20150603214016-ca53cad383ca // indirect
+	github.com/peterbourgon/diskv v2.0.1+incompatible // indirect
+	github.com/philhofer/fwd v1.0.0 // indirect
+	github.com/pquerna/ffjson v0.0.0-20181028064349-e517b90714f7 // indirect
+	github.com/projectcalico/go-json v0.0.0-20161128004156-6219dc7339ba // indirect
+	github.com/projectcalico/go-yaml v0.0.0-20161201183616-955bc3e451ef // indirect
+	github.com/projectcalico/go-yaml-wrapper v0.0.0-20161127220527-598e54215bee
+	github.com/prometheus/client_golang v0.0.0-20171005112915-5cec1d0429b0
+	github.com/prometheus/client_model v0.0.0-20170216185247-6f3806018612 // indirect
+	github.com/prometheus/common v0.0.0-20171104095907-e3fb1a1acd76 // indirect
+	github.com/prometheus/procfs v0.0.0-20171017214025-a6e9df898b13 // indirect
+	github.com/satori/go.uuid v1.2.0
+	github.com/sirupsen/logrus v1.4.0
```

Comment:
Out of interest, is there a syntax for saying "<this> is the package name, but <that> (something else) is the repo to get for that package name"?  I guess that's what we need if we need to reference our logrus fork here.  @fasaxc ?

Reply:
Yes, there's a 'replace' directive. For example:
`replace github.com/sirupsen/logrus => github.com/projectcalico/logrus v1.0.4-calico`